Makefile

```diff
@@ -118,9 +118,13 @@ dialyzer-install:
 dialyzer:
 	@./rebar3 dialyzer
 
-test:
+test: test-app-integration
 	@./rebar3 as test do release, ct $(CT_TEST_FLAGS) --sys_config config/test.config
 
+test-app-integration:
+	@./rebar3 ct $(CT_TEST_FLAGS) --dir test/*/
```

Comment:
It is not clear to me why a separate `rebar3 ct` invocation needed. I shall try moving the test to a new test-only app before approving.

Reply:
I see this is actually documented in https://www.pivotaltracker.com/story/show/154363216

Reply:
@happi I am trying to understand the need for separate `rebar3 ct` call. May you please have a look at https://github.com/aeternity/epoch/pull/612 ?